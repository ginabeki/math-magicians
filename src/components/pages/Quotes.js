import React from 'react';

const Quotes = () => (
  <div style={{
    display: 'flex',
    lineHeight: '1.4',
    justifyContent: 'center',
    alignItems: 'center',
    padding: '2rem',
    background: '#000',
    opacity: '0.8',
    color: '#fff',
    margin: '5rem auto',
    width: '65%',
  }}
  >
    <p>
      Today&apos;s scientists have substituted mathematics for experiments,
      and they wander off through equation after equation, and eventually
      build a structure which has no relation to reality. -- Nikola Tesla
    </p>
  </div>
);

export default Quotes;
