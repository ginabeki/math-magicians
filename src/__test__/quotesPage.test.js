import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/pages/Quotes';

it('renders home page correctly', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
