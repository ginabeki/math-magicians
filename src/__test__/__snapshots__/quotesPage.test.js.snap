// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "background": "#000",
      "color": "#fff",
      "display": "flex",
      "justifyContent": "center",
      "lineHeight": "1.4",
      "margin": "5rem auto",
      "opacity": "0.8",
      "padding": "2rem",
      "width": "65%",
    }
  }
>
  <p>
    Today's scientists have substituted mathematics for experiments, and they wander off through equation after equation, and eventually build a structure which has no relation to reality. -- Nikola Tesla
  </p>
</div>
`;
